actions/java-gradle-setup/action.yaml

```diff
@@ -0,0 +1,33 @@
+name: "Setup gradle"
+description: "Setup java and gradle"
+
+inputs:
+  java-distribution:
+    description: "Java distribution to be installed. (Default is temurin)"
+    required: false
+    default: "temurin"
+  java-version:
+    description: "Java version to be installed. (Default is 11)"
+    required: false
+    default: "11"
+  gradle-version:
+    description: "Gradle version to be installed. (Default is 6.7.1)"
+    required: false
+    default: "6.7.1"
+  gradle-cache:
```

Comment:
I don't see why we need this at all. This is **not** the gradle cache, but the java cache. `mvn` and `sbt` are useless because we don't use them, and gradle's caching is handled by `gradle/gradle-build-action@v2`